### PR TITLE
try populating instruments from localStorage when offline

### PR DESCRIPTION
**Pull request by settinghead**

### CODE REVIEW

File: rise-financial-es6.js [hunk 1]
```diff
@@ -336,6 +350,11 @@
         }
       }
 
+      connectedRef = firebase.database().ref( ".info/connected" );
+      connectedRef.on( "value", ( function onConnectionStateChanged( snap ) {
+        this.firebaseConnected = snap.val();
```
Comment (stulees) [inline rise-financial-es6.js]: @settinghead I think `firebaseConnected` should be added to "private" vars here - https://goo.gl/9oj1Fl and default to `true`. Also can you use the usual _ prefix,  `this._firebaseConnected`.

File: rise-financial-es6.js [hunk 2]
```diff
@@ -336,6 +350,11 @@
         }
       }
 
+      connectedRef = firebase.database().ref( ".info/connected" );
+      connectedRef.on( "value", ( function onConnectionStateChanged( snap ) {
```
Comment (stulees) [inline rise-financial-es6.js]: @settinghead You can use es6 `=>` here instead of `function`

**Review by stulees — CHANGES_REQUESTED** [review]
@settinghead As well as corrections, there are missing deployment files. You need to run `gulp bump` and then run `npm run build`.

**Comment by settinghead** [comment]
@stulees code updated. Thanks.

**Comment by stulees** [comment]
@settinghead Any test coverage possible? Thanks

**Comment by settinghead** [comment]
@stulees added an integration test case.

**Review by stulees — APPROVED** [review]

**Comment by stulees** [comment]
@settinghead Looks good 👍

**Pull request merged**